refactor/runner/openapi.yaml

```diff
@@ -0,0 +1,176 @@
+openapi: 3.0.2
+info:
+  title: Covalent Runner Service API
+  version: 0.1.0
+paths:
+  /api/v0/workflow/{dispatch_id}/tasks:
+    post:
+      tags:
+        - Task
+      summary: Run Tasks
+      description: API Endpoint (/api/workflow/{dispatch_id}/tasks) to run tasks
+      operationId: run_tasks_api_v0_workflow__dispatch_id__tasks_post
+      parameters:
+        - required: true
+          schema:
+            title: Dispatch Id
+            type: string
+          name: dispatch_id
+          in: path
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/TaskPickleList'
+        required: true
+      responses:
+        '202':
+          description: Successful Response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/RunTaskResponse'
+        '422':
+          description: Validation Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/HTTPValidationError'
+  /api/v0/workflow/{dispatch_id}/task/{task_id}:
+    get:
+      tags:
+        - Task
+      summary: Check Status
+      description: API Endpoint (/api/workflow/{dispatch_id}/task/{task_id}) to check status of a task
+      operationId: check_status_api_v0_workflow__dispatch_id__task__task_id__get
+      parameters:
+        - required: true
+          schema:
+            title: Dispatch Id
+            type: string
+          name: dispatch_id
+          in: path
+        - required: true
+          schema:
+            title: Task Id
+            type: integer
```

Comment:
We should be consistent with the type of task IDs, it's changed to `string` in the `POST` call.

Reply:
ok cool that makes sense... and how is the id, function, input, etc represented in the string?

Reply:
All of this will be 1 pickle object which is bytes, it says "string" but its actually a bytes type.